tests/readme_test.rs

```diff
@@ -12,36 +12,30 @@ mod tests {
         dotenv::from_path(Path::new(".env.local")).ok();
         let api_key = env::var("API_KEY").expect("API_KEY not found");
 
-        let mut client = Client::new(&api_key).await;
+        let mut client = Client::new(&api_key).await.unwrap();
         let data = client
             .query("SELECT * FROM eth.recent_blocks LIMIT 10;")
             .await;
         if data.is_err() {
-            assert!(false, "failed to query: {:#?}", data.expect_err(""))
```

Comment:
I don't know if we want to panic here, subsequent tests are still viable if this one fails.